### PR TITLE
add confirm to reboot

### PR DESCRIPTION
**Pull request by gb-hacktoberfest**
Proposed feature for #185

### CODE REVIEW

**Comment by gb-hacktoberfest** [comment]
test broke on circle compared to local. will update

File: tests/test_menu_launcher.py [hunk 1]
```diff
@@ -270,6 +270,16 @@ def test_running_menu():
     # go to system commands
     child.sendline('5')
     child.expect('Return to Vent menu')
+    # choose reboot
+    child.sendline('4')
+    child.expect('Do you want to continue [Y/n]?')
```
Comment (cglewis) [inline tests/test_menu_launcher.py]: your tests might be happier if you make this slightly less specific.  curses on circleci is odd.

**Comment by gb-hacktoberfest** [comment]
@cglewis updated the commit to be circle-ci happier

**Comment by gb-hacktoberfest** [comment]
@cglewis done

**Pull request merged**